### PR TITLE
feat: Added display instructions for shortcuts

### LINKED ISSUE #419: [FEATURE] Display instructions for shortcuts
Tasks for this:
- [ ] for the input box when chatting, replace the placeholder with `press ctrl + enter to send message`
https://github.com/Dun-sin/Whisper/blob/c7c0e4ddafcf7babe12d2f22ccd3ccbdfb2f1ec2/client/src/components/Chat.jsx#L616-L622
- [ ] on the drop-down when chatting, for the close chat add the shortcut to the name which is `ctrl + shift + x`, add an element like this `div(parent element) > p(function e.g close chat) + p(shortcut)` with a tailwind class for the div of `flex space-between items-center`
https://github.com/Dun-sin/Whisper/blob/ab1a0caa53579a1eb931aa319b2f5ecd32987561/client/src/components/Anonymous.jsx#L211-L217
- [ ] for the second content of the drop-down, repeat the `div > p + p` format, but the shortcut for the find a new buddy is `ctrl + alt + n`
https://github.com/Dun-sin/Whisper/blob/ab1a0caa53579a1eb931aa319b2f5ecd32987561/client/src/components/Anonymous.jsx#L215-L217

### Are you willing to work on this issue ?

None

### PR DESCRIPTION
**Pull request by avibiradar100**
# Closes Issue

**My PR closes #419**

# 👨‍💻 Changes proposed (What did you do?)

- while chatting displayed the appropriate placeholder
- Added display shortcuts in the dropdown

# ✔️ Check List (Check all the applicable boxes)

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.

# 📷 Screenshots
Screenshot 2023-10-21 at 12 58 31 PM

<br> 
Screenshot 2023-10-21 at 1 23 34 PM

### CODE REVIEW

File: client/src/components/Chat/MessageInput.jsx [hunk 1]
```diff
@@ -31,11 +34,13 @@ const MessageInput = ({
 			<form className="flex justify-center items-center mt-[40px]" onSubmit={handleSubmit}>
 				<div className="w-full flex items-center justify-between bg-secondary rounded-l-md max-h-[150px] relative">
 					<textarea
-						placeholder="Send a Message....."
+						placeholder={focusTextArea ? 'Press Ctrl + Enter to send message':'Type a message...'}
```
Comment (Dun-sin) [inline client/src/components/Chat/MessageInput.jsx:37]: Just the text, no need for a state

File: client/src/components/Chat/MessageInput.jsx [hunk 2]
```diff
@@ -31,11 +34,13 @@ const MessageInput = ({
...
 						className="h-[45px] focus:outline-none w-[96%] bg-secondary text-white rounded-[15px] resize-none pl-[22px] pr-[22px] py-[10px] text-[18px] placeholder-shown:align-middle min-h-[40px] max-h-[100px] overflow-y-scroll"
 						ref={inputRef}
 						value={message}
 						onChange={handleTypingStatus}
+						onFocus={() => setFocusTextArea(true)}
```
Comment (Dun-sin) [inline client/src/components/Chat/MessageInput.jsx:42]: So this and the blur with the state should be removed

**Comment by avibiradar100** [comment]
Hey @Dun-sin Greetings,
This is my first open source contribution. Hope this PR looks all good. If there is any issue please inform me so that I can rework on it.
Thank You.

File: client/src/components/Anonymous.jsx [hunk 3]
```diff
@@ -205,9 +205,19 @@ const Anonymous = ({ onChatClosed }) => {
 				</div>
 
 				<Dropdown placement="leftStart" style={{ zIndex: 3 }} renderToggle={MenuToggle} noCaret>
-					<Dropdown.Item onClick={() => handleClose()}>Close Chat</Dropdown.Item>
-
-					<Dropdown.Item onClick={() => handleClose(true)}>Find a new buddy</Dropdown.Item>
+					<Dropdown.Item onClick={() => handleClose()}>
+                        <div className="flex items-center space-between">
```
Comment (Dun-sin) [inline client/src/components/Anonymous.jsx:209]: Sorry for the mistake pls change `space-between` to `justify-between`

File: client/src/components/Anonymous.jsx [hunk 4]
```diff
@@ -205,9 +205,19 @@ const Anonymous = ({ onChatClosed }) => {
...
+                        <p>Close Chat</p>
+                        <p className="text-gray-500 text-xs pl-2">Ctrl + Shift + X</p>
```
Comment (Dun-sin) [inline client/src/components/Anonymous.jsx:211]: No need for the padding here

**Review by Dun-sin — CHANGES_REQUESTED** [review]

File: client/src/components/Chat/MessageInput.jsx (hunk 1 above)
Comment (avibiradar100) [reply client/src/components/Chat/MessageInput.jsx]: @Dun-sin  addressed all the comments , thanks for the review 😃

File: client/src/components/Anonymous.jsx [hunk 5]
```diff
@@ -205,9 +205,19 @@ const Anonymous = ({ onChatClosed }) => {
 				</div>
 
 				<Dropdown placement="leftStart" style={{ zIndex: 3 }} renderToggle={MenuToggle} noCaret>
-					<Dropdown.Item onClick={() => handleClose()}>Close Chat</Dropdown.Item>
-
-					<Dropdown.Item onClick={() => handleClose(true)}>Find a new buddy</Dropdown.Item>
+					<Dropdown.Item onClick={() => handleClose()}>
+                        <div className="flex items-center justify-between">
```
Comment (Dun-sin) [inline client/src/components/Anonymous.jsx:209]:
```suggestion
                        <div className="flex items-center justify-between gap-2 flex-wrap">
```

File: client/src/components/Anonymous.jsx [hunk 6]
```diff
@@ -205,9 +205,19 @@ const Anonymous = ({ onChatClosed }) => {
...
+                        <p>Close Chat</p>
+                        <p className="text-gray-500 text-xs">Ctrl + Shift + X</p>
+                        </div>
+                    </Dropdown.Item>
+
+                    <Dropdown.Item onClick={() => handleClose(true)}>
+                        <div className="flex items-center justify-between">
```
Comment (Dun-sin) [inline client/src/components/Anonymous.jsx:216]:
```suggestion
                        <div className="flex items-center justify-between gap-2 flex-wrap">
```

File: client/src/components/Chat/MessageInput.jsx [hunk 7]
```diff
@@ -31,7 +31,7 @@ const MessageInput = ({
 			<form className="flex justify-center items-center mt-[40px]" onSubmit={handleSubmit}>
 				<div className="w-full flex items-center justify-between bg-secondary rounded-l-md max-h-[150px] relative">
 					<textarea
-						placeholder="Send a Message....."
+						placeholder="Press Ctrl + Enter to send message"
```
Comment (Dun-sin) [inline client/src/components/Chat/MessageInput.jsx:34]:
```suggestion
						placeholder="Press Ctrl + Enter to send a message"
```

**Review by Dun-sin — APPROVED** [review]

**Comment by Dun-sin** [comment]
You did great, thanks for contributing, I hope you will stick around and continue to contribute to this project.

Consider giving this project a star and joining the community discord server if you haven't for more resources and opportunities to connect with others. [👉🏽here👈🏽](https://discord.com/invite/ufcysW9q23)

**Pull request merged**